File: sources/com/lumiyaviewer/lumiya/ui/objects/_$Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y.java
/*
 * Decompiled with CFR 0.152.
 */
package com.lumiyaviewer.lumiya.ui.objects;

import com.lumiyaviewer.lumiya.react.Subscription;
import com.lumiyaviewer.lumiya.slproto.users.manager.ObjectsManager;
import com.lumiyaviewer.lumiya.ui.objects.ObjectSelectorFragment;

/*
 * Illegal identifiers - consider using --renameillegalidents true
 */
final class _$Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y
implements Subscription.OnData {
    private final Object -$f0;

    private final /* synthetic */ void $m$0(Object object) {
        ((ObjectSelectorFragment)this.-$f0).lambda$-com_lumiyaviewer_lumiya_ui_objects_ObjectSelectorFragment_10971((ObjectsManager.ObjectDisplayList)object);
    }

    public /* synthetic */ _$Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y(Object object) {
        this.-$f0 = object;
    }

    public final void onData(Object object) {
        this.$m$0(object);
    }
}

File: organized-repos/kotlin-clean/ui/ui/objects/$Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y.kt
package com.linkpoint.ui.objects
import java.util.*

import com.linkpoint.react.Subscription
import com.linkpoint.slproto.users.manager.ObjectsManager

/* renamed from: com.lumiyaviewer.lumiya.ui.objects.-$Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y  reason: invalid class name */
final /* synthetic */ class $Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y : Subscription.OnData {

    /* renamed from: -$f0  reason: not valid java name */
    private val /* synthetic */ Object f499$f0

    private val /* synthetic */ Unit $m$0(Object obj) {
        ((ObjectSelectorFragment) this.f499$f0).m683lambda$com_lumiyaviewer_lumiya_ui_objects_ObjectSelectorFragment_10971((ObjectsManager.ObjectDisplayList) obj)
    }

    public /* synthetic */ $Lambda$rXtKRyOts6GGB3GxWNYA5oEvU2Y(Object obj) {
        this.f499$f0 = obj
    }

    val Unit onData(Object obj) {
        $m$0(obj)
    }
}
